### PR TITLE
adapt to new XML dynamic list format

### PR DESCRIPTION
**Pull request by ammen99**
The autostart and command plugins are still hardcoded, and this PR does not really add support for dynamic lists. This should serve as an temporary solution until such support is implemented.

Tested with autostart and command, adding and removing items works.

### CODE REVIEW

File: src/pages.cpp [hunk 1]
```diff
@@ -1993,13 +2103,7 @@ static void setup_command_list(GtkWidget *widget, Option *o)
...
             }
 
-            if (std::string(o->default_value.s) == "string")
-            {
-                dyn_opt->type = OPTION_TYPE_STRING;
-            } else
-            {
-                continue;
-            }
+            dyn_opt->type = OPTION_TYPE_STRING;
```
Comment (soreau) [inline src/pages.cpp:2106]: What is the reason behind this change?

File: src/pages.cpp [hunk 2]
```diff
@@ -2280,7 +2384,7 @@ static void add_option_widget(GtkWidget *widget, Option *o)
         break;
 
       case OPTION_TYPE_DYNAMIC_LIST:
-        if (std::string(o->name) == "command")
+        if (std::string(o->name) == "bindings")
```
Comment (soreau) [inline src/pages.cpp:2387]: Same question here, why is this being changed?

**Review by soreau — APPROVED** [review]
This appears to be unintrusive but I'm a bit confused about some of the changes.

File: src/pages.cpp (hunk 2 above)
Comment (ammen99) [reply src/pages.cpp:2387]: Because the XML was also renamed. Now we have three different groups, `bindings`, `repeatable_bindings` and `always_bindings`. They don't need separate handling in WCM however (it all works out when rebuilding the values with the prefixes).

File: src/pages.cpp (hunk 1 above)
Comment (ammen99) [reply src/pages.cpp:2106]: There are no default values for dynamic lists, so this check is meaningless.
